ci-operator/step-registry/aws/c2s/init-token-service/aws-c2s-init-token-service-commands.sh

```
@@ -76,27 +78,43 @@ temp_cred_file=$(mktemp)
 
 # request credential must be in enmulator env.
 source "${SHARED_DIR}/proxy-conf.sh"
+
+# Disable errexit so a non-zero curl/jq exit does not abort the script at the
+# command substitution; the retry loop below handles failures itself.
+set +o errexit
 while ([ X"${key_id}" == X"" ] || [ X"${key_id}" == X"null" ]) && [ $try -lt $retries ]; do
```

Shellcheck (0.11.0)

[style] Use { ..; } instead of (..) to avoid subshell overhead.

(SC2235)